### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by tmcw** [mapbox/mts-data-sync#12]
The installation instructions for this utility seem odd relative to other Node-based utilities. Typically I'd expect to see something like

```
npm install -g mts-data-sync
```

To install a global utility, rather than cloning and linking. That would give the utility the benefit of only installing on tagged releases rather than whatever is in the main branch of the repository. Is there some reason why this repository uses a different technique?

**Comment by jbranigan** [comment]
I agree that this should be an NPM package. Modified the title of this issue to reflect that intent.

**Comment by james-innes** [comment]
For the other API endpoints there is a section in the docs telling you which SDK's have been made for it. For example geocoding. However for the tilesets API docs instead of referencing the generic JavaScript driver it points here to a NodeJs only (not browser) driver that seems a little clunky given you can't just `npm install`  it. However the tilesets service does seem to be exposed in the generic Javascript SDK and [documented](https://github.com/mapbox/mapbox-sdk-js/blob/main/docs/services.md#createtilesetsource) here. So unless I'm missing something then most people would probably prefer to use that than this repo.
